ansible/roles/monasca/templates/monasca-log-transformer/log-transformer.conf.j2

```diff
@@ -14,7 +14,7 @@ input {
 filter {
     # Update the timestamp of the event based on the time in the message.
     date {
-        match => [ "[log][dimensions][timestamp]", "yyyy-MM-dd HH:mm:ss +0000", "ISO8601"]
+        match => [ "[log][dimensions][timestamp]", "yyyy-MM-dd HH:mm:ss Z", "ISO8601"]
         remove_field => [ "[log][dimensions][timestamp]", "[log][dimensions][Timestamp]" ]
```

Comment:
Same comment as upstream about this bit - is it not working?

Reply:
Removed log-metrics change to match upstream.